website/templates/emails/pending_retraction_non_admin.txt.mako

```diff
@@ -0,0 +1,7 @@
+Hello ${user.fullname},
+
+I just wanted to let you know, ${initiated_by} has requested a retraction for the following registration: ${registration_link}.
+
```

Comment:
This email doesn't make clear whether or not action is required by the user. Perhaps more clarification is needed. Thoughts, @bnosek ?